modules/nf-core/smoove/main.nf

```diff
@@ -0,0 +1,35 @@
+process SMOOVE {
+    tag "$meta.id"
+    label 'process_single'
```

Comment:
Is `process_single` the best fitting label here since the module can use multiple cores?

Reply:
Good point. I changed to process_high. I'm going to wait for the test to run (and crash) before pushing the changes.

Reply:
Awesome!